packages/calling/src/CallingClient/calling/call.ts

```diff
@@ -1972,8 +1965,9 @@ export class Call extends Eventing<CallEventTypes> implements ICall {
   };
 
   /**
-   * @param settings
-   * @param settings.localAudioTrack
+   * Answers the call using the provided local audio stream.
+   *
+   * @param localAudioStream - The local audio stream for the call.
```

Comment:
Why are we making changes in this file if we decided for the call class to not be exported?